crates/rolldown/tests/esbuild/default/exports_and_module_format_common_js/artifacts.snap

```diff
@@ -11,21 +10,21 @@ snapshot_kind: text
 const node_assert = __toESM(require("node:assert"));
 
 //#region foo/test.js
-var test_exports$1 = {};
-__export(test_exports$1, { foo: () => foo });
+var test_exports = {};
```

Comment:
Since we marked some import statements as included, they participate in the deconflicting and make some symbols to be deconflicted early than they used to be.